### PR TITLE
Task/762  show all users mapped with role issues has completed

### PR DESCRIPTION
**Pull request by jananikuruntham**

### CODE REVIEW

File: api/src/roles/roles.controller.ts [hunk 1]
```diff
@@ -167,4 +168,38 @@ export class RolesController {
...
+  @ApiResponse({
+    status: 500,
+    description: 'Internal server error',
+  })
+  @ApiOperation({ summary: 'Get users by role' })
+  @Put('users/:id')
+  @UsePipes(new ValidationPipe())
+  @HttpCode(HttpStatus.OK)
+  @UseGuards(AuthGuard('jwt'))
+  async updateRole(
```
Comment (vsaravanakumar) [inline api/src/roles/roles.controller.ts:200]: Change the function name

File: api/src/roles/roles.service.ts [hunk 2]
```diff
@@ -216,4 +218,48 @@ export class RolesService {
       );
     }
   }
+  async getRolesByUsers(id: number): Promise<RolesInUserUpdateSuccessDto> {
```
Comment (vsaravanakumar) [inline api/src/roles/roles.service.ts:221]: Change the function name

File: api/src/roles/roles.controller.ts (hunk 1 above)
Comment (jananikuruntham) [reply api/src/roles/roles.controller.ts]: function name was changed

File: api/src/roles/roles.service.ts (hunk 2 above)
Comment (jananikuruntham) [reply api/src/roles/roles.service.ts]: function name was changed

File: front/src/app/roles/[id]/page.tsx [hunk 3]
```diff
@@ -0,0 +1,526 @@
...
+  const handleAddUserClick = () => {
+    setAddUserModalOpen(true);
+  };
+
+  const handleCloseAddUserModal = () => {
+    setAddUserModalOpen(false);
+  };
+
+  const handleAddUser = (newUser: RowData) => {
+    addUser({ ...newUser, role: roleData?.name });
```
[the remark this answers is not in the document]
Comment (jananikuruntham) [reply front/src/app/roles/[id]/page.tsx]: function call was removed

**Pull request merged**